### PR TITLE
Update en.json

### PR DESCRIPTION
**Pull request by CyrilAtHome**
Test 02/06/2020 23:30
Remplacement de "Full Name" par "Nom complet"

### CODE REVIEW

**Comment by CyrilAtHome** [comment]
Ceci est un test pour vérifier si la modification du fichier en.json va avoir un impact sur mon master.

File: lang/en.json [hunk 1]
```diff
@@ -502,7 +502,7 @@
     "forms": {
         "range": "You need to enter numbers only between: {limitTo} and {limitFrom}",
         "contact_us": {
-            "full_name": "Full Name",
+            "full_name": "Nom complet",
```
Comment (junedkazi) [inline lang/en.json:505]: @CyrilAtHome did you mean to modify `fr.json` file ?

Comment (CyrilAtHome) [reply lang/en.json:505]: Hi @junedkazi , till now i just want to test if a modification of en.json has a real impact on my website at bigcommerce.com.
But perhaps i didn't undestand the purpose of github.com :(

Comment (CyrilAtHome) [reply lang/en.json:505]: OK, after reading the Guide, i understood that Github is a platform for learning how to develop correctly.
I have a website on bigcommerce.com, and i have to localize it by modifying fr.json, but i didn't understand how to push it on bigcommerce.com. It seems that i have to install stencil CLI, but the doc on bigcommerce.com was not update with the last version of all modules needed and the install doesn't work.

**Comment by junedkazi** [comment]
I am going to close this one.

**Pull request closed without merging**